src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingFarmer.java

```diff
@@ -728,5 +709,23 @@ public boolean isRecipeCompatible(@NotNull final IGenericRecipe recipe)
             if (!super.isRecipeCompatible(recipe)) return false;
             return CraftingUtils.isRecipeCompatibleBasedOnTags(recipe, FARMER).orElse(false);
         }
+
+        @Override
+        public boolean canRecipeBeAdded(@NotNull final IToken<?> token)
+        {
+            if (!super.canRecipeBeAdded(token))
+            {
+                return false;
+            }
+            Optional<Boolean> isRecipeAllowed;
+
+            if (!super.canRecipeBeAdded(token))
+            {
+                return false;
+            }
+
+            isRecipeAllowed = super.canRecipeBeAddedBasedOnTags(token);
+            return isRecipeAllowed.orElse(false);
+        }
```

Comment:
This looks like a merge derp or something, calling super method twice is silly and calling `canRecipeBeAddedBasedOnTags` happens in the super method anyway.  So it can probably be removed entirely, like most of the other crafters.

Reply:
True, I'll remove it from here.